js/interactions.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+let $ = require("jquery"),
+    classes = require("./robotClasses"),
+    selection = require("./robotSelection");
+
+let playerName1 = "",
+    playerRobot1 = null,
+    playerName2 = "",
+    playerRobot2 = null;
+
+// name and type selected bot 1
+  $(".submit1").click(function(e) {
+
+  playerName1 = $("#player1input").val();
+  $('.player1display').html(playerName1);
+  console.log(playerName1);
+  playerRobot1 = $("#playerRobot1").val();
+  console.log(playerRobot1);
```

Comment:
Here is where you can instantiate a new robot. But, first you will need to fix the circular module require statements, because if you log "classes" or "selection" you'll see that they are empty objects. Once you have those objects populated with the exported functions, you can then instantiate a robot using the `playerRobot1` variable. "classes" ill contain all those robots functions, so you can access the one you need with bracket notation, which allows us to access a property using a variable. `let robot1 = new classes[playerRobot1]()`. Of course, the `playerRobot1` var will have to match the name of the function _exactly_.